protocol/js/flow-types-chat.js

```diff
@@ -16,6 +16,60 @@ export type RPCError = {
   code: number,
   desc: string
 }
+export type WaitingHandlerType = (waiting: boolean, method: string, sessionID: number) => void
+
+// $FlowIssue we're calling an internal method on engine that's there just for us
+const engineRpcOutgoing = (...args) => engine._rpcOutgoing(...args)
```

Comment:
i buy controlling when its constructed. we can force you to do engine() like you suggest. i'm just saying we're not building multiple engines